loguru.cpp

```diff
@@ -1792,4 +1797,10 @@ namespace loguru
 
 #endif // _WIN32
 
+#ifdef _WIN32
```

Comment:
seems like the above `#endif // _WIN32` and this `#ifdef _WIN32` cancel each other out?

Reply:
@emilk Have you had a chance to review my comment yet? Thanks!